.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+name: Test and build
+
+on: [push]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: ['ubuntu-latest', 'macos-latest']
+        # os: ['ubuntu-20.04', 'ubuntu-18.04', 'ubuntu-16.04', 'macos-latest']
+      fail-fast: false
+    runs-on: ${{ matrix.os }}
+      
+    steps:
+    - name: Install Dependencies (Linux)
+      run: |
+        sudo apt-get update -y
+        sudo apt-get install -y devscripts debhelper moreutils fakeroot jq pigz
+        wget "https://storage.googleapis.com/shellcheck/shellcheck-v0.7.0.linux.x86_64.tar.xz"
+        tar --xz -xvf "shellcheck-v0.7.0.linux.x86_64.tar.xz"
+        sudo cp shellcheck-v0.7.0/shellcheck /usr/bin/shellcheck
+      if: matrix.os != 'macos-latest'
```

Comment:
how about 
```
if: matrix.os == 'ubuntu-latest'
```
which is consistent with the `if: matrix.os == 'macos-latest'` below

Reply:
Yeah, makes sense for now. But we will need to change it back, if will be adding older ubuntu versions in testing (once flaky test fixed for them)